comment/api/permissions.py

```diff
@@ -36,3 +36,18 @@ def has_permission(self, request, view):
 
     def has_object_permission(self, request, view, obj):
         return obj.is_flagged and (is_comment_admin(request.user) or is_comment_moderator(request.user))
+
+
+class SubscriptionEnabledPermission(permissions.BasePermission):
+    """
+    This will check if the COMMENT_ALLOW_SUBSCRIPTION is enabled
+    """
+    def has_permission(self, request, view):
+        return getattr(settings, 'COMMENT_ALLOW_SUBSCRIPTION', False)
```

Comment:
I also don't quite understand why we use default values in our codebase with `getattr` when we already initialize these attributes in the `defaults.py` file.

Reply:
Maybe following the good practice? 😄

Reply:
seems unnecessary duplication of code to me. also, probably defeats the overall purpose of having a `defaults` module.

Reply:
I am not sure how you get rid of the duplication when you use this `settings.COMMENT_ALLOW_SUBSCRIPTION` instead of this `getattr(settings, 'COMMENT_ALLOW_SUBSCRIPTION', False)` ?
If we will have a comment on each line for our preferences we will not finish this feature in this holiday. 😩